cumulus_library/actions/builder.py

```diff
@@ -437,7 +437,7 @@ def _check_if_preparable(prefix):
     # This list should include any study which requires interrogating the database to
     # find if data is available to query (outside of a toml-driven workflow),
     # which isn't doable as a distributed query
-    if prefix in ["core", "discovery", "data-metrics"]:
+    if prefix in ["core", "discovery", "data_metrics", "example_nlp"]:
```

Comment:
This bit right here is interesting to me. How important is prepare mode? I'm imagining most studies that do post-processing on NLP results are gonna want to do a similar thing - only run that part of the study if the NLP results are sitting there. So this list will grow, likely.

Reply:
We talked about this offline - https://github.com/smart-on-fhir/cumulus-library/issues/395 covers the details about why we need to be concerned about prepare mode here.